### ISSUE TITLE
[Bug] .bashrc, etc may be updated in the wrong order

### ISSUE THREAD

**Issue by willnode** [webinstall/webi-installers#862]
## What were you trying to install (or what else went wrong)?

Pyenv

## What exactly did you do?

```
curl -sS https://webinstall.dev/pyenv | bash
```

then reload the terminal.

## What went wrong?

My terminal shows up like this

```
Last login: Mon Jul 15 00:20:31 2024 from ::1
-bash: pyenv: command not found
-bash: pyenv: command not found
[todo-app@pc ~]$ 
```

Because in .bashrc these two were flipped

```
# added by Webi for pyenv
eval "$(pyenv init -)"
eval "$(pyenv virtualenv-init -)"

# Generated for envman. Do not edit.
[ -s "$HOME/.config/envman/load.sh" ] && source "$HOME/.config/envman/load.sh"
```

Reordering this manually fixes it, as a workaround.

## Which OS did you try on?

- [ ] Mac
- [ ] Windows
- [X] Linux

## What type of computer (i.e. laptop, desktop, Raspberry Pi)?

- [X] Laptop or Desktop (amd64 / x86-64)
- [ ] Raspberry Pi (ARM-64)
- [ ] Other (please specify, if you know)

**Comment by coolaj86** [comment]
Ah, this appears to be an order-of-operations issue.

There are pre-install steps and post-install steps and pyenv's post-install steps are happening before webi's post-install steps.

If you've used `webi` previously then it would work as expected, but when `pyenv` is the first install, the generic `post-install` steps happen _after_ the `pyenv`-specific post-install steps.

I'll see if I can find an easy way to switch some of the post-install steps to be pre-install scripts.

Thanks for the report.

**Comment by coolaj86** [comment]
@willnode This is fixed in #863 and already deployed as a hotfix.

**Comment by willnode** [comment]
Thank  you!
